lua/entities/acf_base_scalable/init.lua

```diff
@@ -83,12 +102,16 @@ do -- Entity linking and unlinking --------------
 		if self == Target then return false, "Can't unlink an entity from itself." end
 
 		local Class    = Target:GetClass()
-		local Function = ACF.GetClassUnlink(self:GetClass(), Class)
+		local LinkData, Reversed = ACF.GetClassLink(self:GetClass(), Class)
+		local Function = LinkData.Unlink
+
+		if LinkData == nil then return false, "Links between these two entities are impossible" end
```

Comment:
If `LinkData` never actually receives a false value as a valid result, then you could just change this condition to `if not LinkData`